common/remote/sockfile.cpp

```diff
@@ -4508,7 +4508,7 @@ void checkExpiryTime(IPropertyTree &metaInfo)
     {
         expiryTimeDt.setString(expiryTime);
     }
-    catch (IException *e)
+    catch (IException * /*e*/)
```

Comment:
No. This is an example of why I am very wary of huge "just fix warnings so it must be safe" pull requests.

The warning that you have suppressed here (e unused) is actually showing up a bug (that e is leaked).

I cannot review a change this large and expect to spot all such issues. It needs to be broken up into manageable chunks, each one fixing a single category of warning that can be verified by inspection or automated compare.

Reply:
Good point. Add the release of e.